lib/net/pool.js

```diff
@@ -3456,20 +3442,29 @@ class Pool extends EventEmitter {
    */
 
   fillOutbound() {
-    const need = this.options.maxOutbound - this.peers.standard.outbound;
-
     if (!this.peers.load)
       this.addLoader();
 
-    if (need <= 0)
+    const {maxOutbound, brontideMaxOutbound} = this.options;
+    const need = maxOutbound - this.peers.standard.outbound;
+    const brontideNeed = brontideMaxOutbound - this.peers.brontide.outbound;
+
+    if (need <= 0 && brontideNeed <= 0)
```

Comment:
We should check each case in a separate if statement. Otherwise, we may print an unnecessary log statement, e.g. `need === 0`, but `brontideNeed === 1`.

Reply:
Added the `if` statements below.